contribs/github-bot/comment.go

```diff
@@ -0,0 +1,314 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"github.com/gnolang/gno/contribs/github-bot/internal/client"
+
+	"github.com/google/go-github/v64/github"
+	"github.com/sethvargo/go-githubactions"
+)
+
+var errTriggeredByBot = errors.New("event triggered by bot")
+
+// Compile regex only once
+var (
+	// Regex for capturing the entire line of a manual check
+	manualCheckLine = regexp.MustCompile(`(?m:^- \[([ x])\] (.+)?$)`)
+	// Regex for capturing only the user who checked it
+	manualCheckDetails = regexp.MustCompile(`(?m:(.+) \(checked by @(\w+)\)$)`)
```

Comment:
Nit: these can be merged, but if you want to leave them like this its OK

Reply:
Merged :)